Views/Home/Index.cshtml

```diff
@@ -274,48 +277,48 @@
+            // $("#saveProjectButton").click(function () {
 
-                var Sources = [];
-                var Targets = [];
+            //     var Sources = [];
+            //     var Targets = [];
 
-                var maxElements = 5;
+            //     var maxElements = 5;
 
-                for (var i = 0; i < maxElements; i++) {
+            //     for (var i = 0; i < maxElements; i++) {
 
-                    var selectSourceId = "projectSelectSource" + i;
-                    var selectTargetId = "projectSelectTarget" + i;
+            //         var selectSourceId = "projectSelectSource" + i;
+            //         var selectTargetId = "projectSelectTarget" + i;
 
-                    var selectSourceValue = $("select[name='" + selectSourceId + "']").val();
-                    var selectTargetValue = $("select[name='" + selectTargetId + "']").val();
+            //         var selectSourceValue = $("select[name='" + selectSourceId + "']").val();
+            //         var selectTargetValue = $("select[name='" + selectTargetId + "']").val();
 
-                    if (selectSourceValue !== undefined) {
-                        Sources.push(selectSourceValue);
-                    }
+            //         if (selectSourceValue !== undefined) {
+            //             Sources.push(selectSourceValue);
+            //         }
 
-                    if (selectTargetValue !== undefined) {
-                        Targets.push(selectTargetValue);
-                    }
-                }
+            //         if (selectTargetValue !== undefined) {
+            //             Targets.push(selectTargetValue);
+            //         }
+            //     }
 
-                var formData = {
-                    Name: $("#projectName").val(),
-                    Version: $("#projectVersion").val(),
-                    Description: $("#projectDescription").val(),
-                    Sources: Sources,
-                    Targets: Targets
-                };
+            //     var formData = {
+            //         Name: $("#projectName").val(),
+            //         Version: $("#projectVersion").val(),
+            //         Description: $("#projectDescription").val(),
+            //         Sources: Sources,
+            //         Targets: Targets
+            //     };
 
-                console.log(formData);
+            //     console.log(formData);
 
-                $.ajax({
-                    data: formData, 
-                    success: function (response) {
-                    },
-                    error: function (error) {
-                    }
-                });
-            });
+            //     $.ajax({
+            //         data: formData, 
+            //         success: function (response) {
+            //         },
+            //         error: function (error) {
+            //         }
+            //     });
+            // });
         });
     </script>
 }
```

Comment:
Missing newline at end of file

Reply:
Does this mean I have to add a new line at the end of each file? In the IDE I use I don't have the notification telling me to add a new line.

Reply:
In principle yes, this is standard at least for actual source code files. Github also shows missing newlines at end of file with a red sign.

Reply:
Ok l will add a newline at the end of each file